### PR TITLE
[22] Newsletter Form Validation

### PR DESCRIPTION
**Pull request by tomike19**
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

**Comment by tomike19** [comment]
Kindly review my PR

File: submissions/kolajoTomike/Javascript/Task1/app.js [hunk 1]
```diff
@@ -20,4 +20,7 @@ var productList=[
 ];
  productList.forEach((product)=>{
     console.log(product.productName + ' with '+ product.imageUrl + ' is sold for ' + product.productPrice)
- });
\ No newline at end of file
+ });
+
```
Comment (adaugochi) [inline submissions/kolajoTomike/Javascript/Task1/app.js:24]: Remove spaces

File: submissions/kolajoTomike/.vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (adaugochi) [inline submissions/kolajoTomike/.vscode/settings.json:1]: Remove this file

**Review by adaugochi — CHANGES_REQUESTED** [review]

**Comment by adaugochi** [comment]
Add a screenshot of the error message you got

File: submissions/kolajoTomike/.vscode/settings.json (hunk 2 above)
Comment (tomike19) [reply submissions/kolajoTomike/.vscode/settings.json]: kindly reveiw again

**Review by adaugochi — APPROVED** [review]

**Pull request merged**